universal-application-tool-0.0.1/conf/evolutions/default/12.sql

```diff
@@ -0,0 +1,8 @@
+# --- Add answer options for multi-option questions
+# --- !Ups
+
+alter table applicants add when_created timestamp;
+
```

Comment:
Suggestion (blocking): would you add a default value for existing entities so that we don't break by null pointer exception?
Something like `update applicants set whenCreated = '<default value here>' where whenCreated is null; ` would do it.